src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="airbnb.com"
+      class="form big-search-form"
+      method="post"
+      data-qa="big"
+    >
+      <label class="field">
+        <img
+          src="./images/Search.svg"
+          alt="search-svg-img"
+          class="img big-search-img"
+        >
+        <input
+          type="search"
+          name="bigSearch"
+          class="search-input big-search-input"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
```

Comment:
```suggestion
      action="airbnb.com"
      class="form"
      method="post"
      data-qa="big"
    >
      <label class="field">
        <img
          src="./images/Search.svg"
          alt="search-svg-img"
          class="img"
        >
        <input
          type="search"
          name="bigSearch"
          class="search-input"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
check 2-nd point in checklist